### PR TITLE
Adds description in bucket class dropdown for OBC page

### PR DESCRIPTION
**Pull request by vbnrh**
Design: https://marvelapp.com/prototype/fa616je/screen/74583172

Screenshot from 2021-04-08 12-09-01

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/object-bucket-claim-page/create-obc.tsx [hunk 1]
```diff
@@ -78,6 +78,24 @@ export const CreateOBCPage: React.FC<CreateOBCPageProps> = (props) => {
...
+  };
+
+  const transformLabel = (resource: K8sResourceKind) => {
+    return (
+      <span className="co-resource-item__resource-name">
+        {getName(resource)}
+        <div className="text-muted small">{getBucketClassDescription(resource)}</div>
+      </span>
+    );
+  };
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/object-bucket-claim-page/create-obc.tsx:98]:
```suggestion
onst transformLabel = (resource: K8sResourceKind) => (
      <span className="co-resource-item__resource-name">
        {getName(resource)}
        <div className="text-muted small">{getBucketClassDescription(resource)}</div>
      </span>
    );
```

File: frontend/packages/ceph-storage-plugin/src/components/object-bucket-claim-page/create-obc.tsx [hunk 2]
```diff
@@ -78,6 +78,24 @@ export const CreateOBCPage: React.FC<CreateOBCPageProps> = (props) => {
...
 
+  const getBucketClassDescription = (resource: K8sResourceKind) => {
+    if (resource.spec?.namespacePolicy) {
+      return `Type: Namespace | Policy: ${resource.spec.namespacePolicy.type}`;
+    }
+    if (resource.spec?.placementPolicy) {
+      return `Type: Standard | Tiers: ${resource.spec.placementPolicy.tiers.length}`;
+    }
+    return '';
+  };
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/object-bucket-claim-page/create-obc.tsx:89]: externalize the strings

**Review by afreen23 — CHANGES_REQUESTED** [review]

File: frontend/packages/ceph-storage-plugin/src/components/object-bucket-claim-page/create-obc.tsx (hunk 2 above)
Comment (vbnrh) [reply frontend/packages/ceph-storage-plugin/src/components/object-bucket-claim-page/create-obc.tsx]: done

File: frontend/packages/ceph-storage-plugin/src/components/object-bucket-claim-page/create-obc.tsx (hunk 1 above)
Comment (vbnrh) [reply frontend/packages/ceph-storage-plugin/src/components/object-bucket-claim-page/create-obc.tsx]: done

**Review by afreen23 — APPROVED** [review]

**Pull request merged**